style.css

```diff
@@ -0,0 +1,328 @@
+body {
+  background-color: lightgoldenrodyellow;
+  font-size: 20px;
+  padding: 0;
+}
+
+p {
+  text-align: justify;
+}
+.container {
+  box-shadow: 5px 5px 5px rgba(122, 122, 122, 0.5);
+  border-radius: 20px;
+  box-sizing: border-box;
+  max-width: 1024px;
+  margin: 0 auto;
+  background-color: linen;
+}
+
+header {
+  border-top-left-radius: 20px;
+  border-top-right-radius: 20px;
+  background-color: #fde4cf;
+  min-height: 400px;
+}
+
+.photo {
+  width: 270px;
+}
+
+.contact_photo_wrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  flex-wrap: wrap-reverse;
+  padding: 10px 10px 10px 20px;
+  align-items: center;
+}
+
+.list {
+  padding-left: 20px;
+}
+
+h1 {
+  font-family: "Jockey One", sans-serif;
+}
+
+h3 {
+  font-family: "Courgette", cursive;
+}
+
+h1,
+h2,
+h3 {
+  text-align: center;
+}
+
+.name {
+  font-size: 50px;
+  text-transform: uppercase;
+  font-weight: bold;
+  text-decoration: underline #66cccc;
+  word-spacing: 0.5em;
+  margin: 0 auto;
+  padding-top: 30px;
+}
+
+.position {
+  background-color: lightgray;
+  max-width: 400px;
+  border-radius: 10px;
+  font-size: 30px;
+  margin: 30px auto;
+}
+
+.contacts {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.contact_link {
+  text-decoration: none;
+  color: black;
+  font-size: 1.1em;
+  padding-top: 10px;
+}
+
+.contact_link:before {
+  content: "";
+  background-size: cover;
+  height: 30px;
+  width: 30px;
+  border-radius: 50%;
+  margin-right: 20px;
+  display: inline-block;
+  vertical-align: middle;
+  margin: 10px 20px 10px 0;
+}
+
+.mail:before {
+  background-image: url(images/mail_icon.png);
+}
+
+.ln:before {
+  background-image: url(images/linked_icon2.png);
+}
+
+.tel:before {
+  background-image: url(images/phone_icon.png);
+}
+
+.gh:before {
+  background-image: url(images/github_icon.png);
+}
+
+.contact_link:hover:before {
+  content: "";
+  background-size: cover;
+  height: 30px;
+  width: 30px;
+  border-radius: 50%;
+  margin-right: 15px;
+  margin-left: 5px;
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.gh:hover:before {
+  background-image: url(images/github_icon.png);
+}
+
+.tel:hover:before {
+  background-image: url(images/phone_icon.png);
+}
+
+.ln:hover:before {
+  background-image: url(images/linked_icon2.png);
+}
+
+.mail:hover:before {
+  background-image: url(images/mail_icon.png);
+}
+
+section {
+  padding: 10px 20px 10px 20px;
```

Comment:
Can be shortened to `padding: 10px 20px;`

Reply:
fix it, thank you